hardware/computemodule/cmio-display.md

```diff
@@ -95,8 +95,12 @@ DISPLAY_DSI_PORT
 The I2C port, SDA and SCL pin numbers are self explanatory. DISPLAY_DSI_PORT
 selects between DSI1 (the default) and DSI0.
 
+Once all the required changes have been made to the ```dts``` file, it needs to be compiled and placed on the boot partition of the device.
+
+Instructions for doing this can be found on the [Pin Configuration](https://github.com/raspberrypi/documentation/blob/master/configuration/pin-configuration.md) page.
+
```

Comment:
this needs to be a relative link so that it works both in github and on the docs site.

Reply:
Doh. Sorry.